fixtures/generic/color-temperature-fader.json

```
@@ -0,0 +1,52 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Color Temperature Fader",
+  "categories": ["Other"],
+  "meta": {
+    "authors": ["Ken"],
```

```suggestion
    "authors": ["Ken Harris"],
```

to match your other fixtures